gcp_variant_transforms/vcf_to_bq.py

```diff
@@ -87,14 +79,17 @@ class PipelineModes(enum.Enum):
 
 def _get_variant_merge_strategy(known_args):
   if (not known_args.variant_merge_strategy or
-      known_args.variant_merge_strategy == _NONE_STRING):
+      known_args.variant_merge_strategy ==
+      variant_transform_options.MergeOptions.NONE_STRING):
```

Comment:
I think you can just create an alias for this in the def.
e.g. merge_options = variant_transform_options.MergeOptions
then do merge_options.NONE_STRING etc